types/stylelint/index.d.ts

```diff
@@ -1,5 +1,5 @@
 declare module 'stylelint' {
-	import {Result, ResultMessage, WarningOptions, Warning} from 'postcss';
+	import {Result, ResultMessage, WarningOptions, Warning, Root as PostcssRoot, NodeSource} from 'postcss';
```

Comment:
[nitpick] New imported `PostcssRoot` and `NodeSource` seems unused...? 🤔

Reply:
I'll revert this change.